app/controllers/api/v1/admin/domain_blocks_controller.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+class Api::V1::Admin::DomainBlocksController < Api::BaseController
+  include Authorization
+  include AccountableConcern
+
+  LIMIT = 100
+
+  before_action -> { authorize_if_got_token! :'admin:read', :'admin:read:domain_blocks' }, only: [:index, :show]
```

Comment:
This is the first multi-word OAuth scope. Should we go with underscores here or use dashes?

Reply:
I'm fine either way.